src/Traits/HasRole.php

```diff
@@ -66,7 +66,7 @@ public function hasRole($role)
             return ($intersectionCount > 0) ? true : false;
         }
 
-        return ! ! $role->intersect($this->roles)->count();
+        return ! ! array_filter($role->only($this->roles))->count();
```

Comment:
Thanks! But I think `count()` should be removed since it's not a collection anymore.

Reply:
I think I made wrong fix. There is no need to change Collections ```intersect``` method, because it wasn't removed in Laravel 5.5. It was Requests method deprecated instead of Collections method, so no changed needed here and you can close pull request, if I'm right.